README.md

```diff
@@ -42,54 +39,60 @@ And sync your Datasources, Components, and stories. it is best to first delete b
 
 ### ⚙️ 5. Add your site to the astro.config and set your adapter (vercel or netlify)
 
-
 ```javascript
 
 export default defineConfig({
-	site: 'https://your-website.com',
-	output: "hybrid",
-  	adapter: vercel(), // vercel() or netlify()
+    site: 'https://your-website.com',
+    output: "hybrid",
+    adapter: vercel(), // vercel() or netlify()
 
     ....
 
 ```
 
+### 🚀 6. Deploy to Vercel or Netlify
+
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Funfolding-io%2FAstrorante&env=STORYBLOK_PREVIEW_TOKEN,STORYBLOK_PERSONAL_TOKEN,STORYBLOK_SPACE_ID,LOCALE,CURRENCY,SITE_LANG&envDescription=All%20ENV%20variables%20are%20mandatory)
+[![Deploy to Netlify](https://www.netlify.com/img/deploy/button.svg)](https://app.netlify.com/start/deploy?repository=https://github.com/Barokai/Astrorante)
```

Comment:
Needs to be changed before merging - currently pointing to the fork to be able to get data from netlify.toml file.
```suggestion
[![Deploy to Netlify](https://www.netlify.com/img/deploy/button.svg)](https://app.netlify.com/start/deploy?repository=https://github.com/unfolding-io/Astrorante)
```

Reply:
Thanks @Barokai!
I will merge this asap.